dask/array/tests/test_cupy_core.py

```diff
@@ -604,3 +604,16 @@ def test_setitem_errs():
     # RHS has extra leading size 1 dimensions compared to LHS
     x = cupy.arange(12).reshape((3, 4))
     dx = da.from_array(x, chunks=(2, 3))
+
+
+@pytest.mark.parametrize("xp", [np, da])
+@pytest.mark.parametrize("array_func", ["array", "asarray", "asanyarray"])
+def test_array_like(xp, array_func):
```

Comment:
Sorry, I should have thought about this earlier, but we'll want to skip these `*_like` tests when NumPy <1.20.0 is installed

Reply:
Thanks @jrbourbeau , I actually had that in my mind and ended up forgetting about it, should be fixed now.